### ISSUE TITLE
Validation dataset split?

### ISSUE THREAD

**Issue by GuHY777** [tiandunx/loss_function_search#2]
Hello! First of all, thank you very much for releasing the code. My question is, is the validation dataset split from the training data? Or directly use the test data set as the validation dataset?

**Comment by tiandunx** [comment]
Thanks for your attention to our work. Well, in our experiments, we use LFW as our validation set.
